src/api/group.messages.ts

```diff
@@ -20,10 +21,12 @@ export function msgCreateGroupWithPolicy(values: GroupWithPolicyFormValues) {
     policyType,
     votingWindow,
   } = values
-  return GroupMsgWithTypeUrl.createGroupWithPolicy({
+  // NOTE: We use the encoded msg type to support amino signing with nested types.
+  // See https://github.com/osmosis-labs/telescope/issues/281
+  const encodedMsg: MsgCreateGroupWithPolicyEncoded = {
     admin,
-    groupPolicyMetadata: '',
-    groupPolicyAsAdmin: policyAsAdmin === 'true',
+    groupPolicyMetadata: ' ', // TODO: unauthorized error when empty
```

Comment:
have you tried with undefined instead (using `clearEmptyStr`)?
I remember we were doing something similar in the amino converters within the ts-proto based regen-js code: https://github.com/regen-network/regen-js/blob/022215439e0db0771e5968d4532fa68630096521/packages/api/src/tx/modules/ecocredit/converter-utils.ts#L7